src/app/calendar-app/event-editor-dialog.component.ts

```diff
@@ -62,6 +77,18 @@ export class EventEditorDialogComponent {
     }
 
     onSubmitClick(): void {
+        this.event.dtstart = moment(this.event.start);
+        this.event.dtend = moment(this.event.end);
+
+        if (this.recurring_frequency === -1) {
+            if (this.event.rrule) {
+                this.event.rrule = undefined;
+            }
+        } else {
+            // FIXME preserve possible existing other settings!
```

Comment:
Is this a reminder for later?

Reply:
Oh, so that's why I didn't submit that PR before... ;)

Thanks for catching it, indeed it is.

Reply:
Fixed now in cf93f06